src/components/About.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Viaduct from './../images/viaduct.png';
+import Quebec from './../images/quebec.png';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFilePdf } from '@fortawesome/free-solid-svg-icons';
+import { faGithub, faLinkedin } from '@fortawesome/free-brands-svg-icons';
+import ScrollingHeadline from './ScrollingHeadline';
+import {
+  Container,
+  Padding,
+  BlobMask,
+  Blobs,
+  SectionHeaderAlt,
+} from './../styles';
+import { Parallax } from 'react-scroll-parallax';
+
+const AboutContainer = styled(Container)`
```

Comment:
I was thinking of this, but wondered if extrapolating them off into their own file would defeat the purpose of CSS-in-JS. Since it's getting a little unwieldy now I agree, prob best to put them in their own file for readability.

Reply:
You'd normally separate these, you still have "css in js" it's just extracted and living in its own file rather than mingling with your component logic. 👍